src/main/java/de/rabea/request/HttpVerb.java

```diff
@@ -15,6 +15,12 @@ public static HttpVerb convert(String word) {
                 return verb;
             }
         }
-        return null;
+        throw new UnknownHttpVerbException("Not a valid Http verb");
```

Comment:
I moved the message into the Exception. I guess in this case it would be a complaint?